### DISCUSSION TITLE
Possible to let Grafana send the timerange defined an dashboard level to the JSON datasource

### DISCUSSION THREAD

**Discussion by dakoller** [grafana/grafana-infinity-datasource#528] [Q&A]
Good day,

I defined a JSON data source, which contains some climate data from a hydroponic tower and the data is taken from the API of another tool --> so the effort (on the API side) to get different scopes of data based on timerange is different.

Is it possible to put the set timerange from Grafana into e.g. a URL Parameter, which can be appended to the JSON url?

Best regards,

Daniel

**Comment by yesoreyeram** [comment]
Yes. I think [this](https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#__from-and-__to) is what you are looking after.
